core/trino-main/src/main/java/io/trino/server/ui/OAuth2WebUiLogoutResource.java

```diff
@@ -50,7 +48,7 @@ public OAuth2WebUiLogoutResource(OAuth2Client auth2Client)
 
     @ResourceSecurity(WEB_UI)
     @GET
-    public Response logout(@Context HttpHeaders httpHeaders, @Context UriInfo uriInfo, @Context SecurityContext securityContext)
```

Comment:
I assume all these contexts being removed has no negative impact and it just a clean up

Reply:
no. the context was used to call a method that delegated to jetty anyway. I just skipped intermediaries

Reply:
Dropped :)